packages/typespec-azure-core/lib/decorators.tsp

```diff
@@ -12,6 +12,11 @@ namespace Azure.Core {
    */
   extern dec pagedResult(entity: Model);
 
+  /**
```

Comment:
should this be going in azure core?

Reply:
Ah I guess it is not at all Azure specific. Would it be better in client-generator-core?

Reply:
I would def put it in tcgc then, because we have more control here and we can always elevate it later if need be. @m-nash is this good with you